include/amici/symbolic_functions.h

```diff
@@ -3,32 +3,242 @@
 
 namespace amici {
 
+/**
+ * C implementation of log function, this prevents returning NaN values for
+ * negative values
+ *
+ * @param x argument
+ * @return if(x>0) then log(x) else -Inf
+ *
+ */
 double log(double x);
+
+/**
+ * C implementation of matlab function dirac
```

Comment:
I would not write "matlab" for these functions, as this is just an implementation of that function, not specific to the matlab side (same for heaviside, min, max, ...).

Reply:
Was just cut-copy-pasting old docstrings. Yep, would also be a good possibility to update them ;)